datahub-web-react/src/app/entity/user/UserProfile.tsx

```diff
@@ -1,110 +1,411 @@
     const entityRegistry = useEntityRegistry();
-    const username = data?.corpUser?.username;
 
+    const username = data?.corpUser?.username;
     const ownershipResult = useGetAllEntitySearchResults({
         query: `owners:${username}`,
     });
 
+    const [groupSectionExpanded, setGroupSectionExpanded] = useState(false);
+    const [editProfileModal, setEditProfileModal] = useState(false);
+    /* eslint-disable @typescript-eslint/no-unused-vars */
+    const [editableAboutMeText, setEditableAboutMeText] = useState<string | undefined>('');
+
+    const groupMemberRelationships = data?.corpUser?.relationships as EntityRelationshipsResult;
+    const groupsDetails = data?.corpUser?.relationships as ExtendedEntityRelationshipsResult;
+    const profileSrc = data?.corpUser?.editableProperties?.pictureLink || undefined;
     const contentLoading =
         Object.keys(ownershipResult).some((type) => {
             return ownershipResult[type].loading;
         }) || loading;
 
-    const ownershipForDetails = useMemo(() => {
-        const filteredOwnershipResult: {
-            [key in EntityType]?: Array<SearchResult>;
-        } = {};
-
-        Object.keys(ownershipResult).forEach((type) => {
-            const entities = ownershipResult[type].data?.search?.searchResults;
-
-            if (entities && entities.length > 0) {
-                filteredOwnershipResult[type] = ownershipResult[type].data?.search?.searchResults;
-            }
-        });
-        return filteredOwnershipResult;
-    }, [ownershipResult]);
-
     if (error || (!loading && !error && !data)) {
         return <Alert type="error" message={error?.message || 'Entity failed to load'} />;
     }
 
-    const groupMemberRelationships = data?.corpUser?.relationships as EntityRelationshipsResult;
-
+    // Routed Tabs Constants
     const getTabs = () => {
         return [
             {
-                name: TabType.Ownership,
-                path: TabType.Ownership.toLocaleLowerCase(),
-                content: <RelatedEntityResults searchResult={ownershipForDetails} />,
+                name: TabType.Assets,
+                path: TabType.Assets.toLocaleLowerCase(),
+                content: <UserAssets />,
+                display: {
+                    enabled: () => true,
+                },
             },
             {
                 name: TabType.Groups,
                 path: TabType.Groups.toLocaleLowerCase(),
                 content: (
                     <UserGroups urn={urn} initialRelationships={groupMemberRelationships} pageSize={GROUP_PAGE_SIZE} />
                 ),
+                display: {
+                    enabled: () => groupsDetails?.relationships.length > 0,
+                },
             },
         ].filter((tab) => ENABLED_TAB_TYPES.includes(tab.name));
     };
+    const defaultTabPath = getTabs() && getTabs()?.length > 0 ? getTabs()[0].path : '';
+    const onTabChange = () => null;
```

Comment:
Why is this needed?

Reply:
The default routeTabs I have utilised, so I have use the same properties.